<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>493</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="LBL_ingestHelp">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_ingest">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ingest</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="footage">
      <attribute name="title">
       <string>Footage</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="RDO_manual">
             <property name="text">
              <string>Manual import</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RDO_edl">
             <property name="text">
              <string>From EDL</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QListWidget" name="LST_files">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="toolTip">
            <string>Each footage imported will be split into any number of shots, which are represented by separate EXR sequences.</string>
           </property>
           <property name="text">
            <string>Shots</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="BTN_shot">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QTableWidget" name="TBL_shots">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="columnCount">
            <number>7</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="BTN_footage">
             <property name="toolTip">
              <string>Import footage to be ingested</string>
             </property>
             <property name="text">
              <string>Import footage...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BTN_edl">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Import EDL...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BTN_rmFootage">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="LBL_fps">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="toolTip">
                <string>FPS of the footage that will be imported by the EDL</string>
               </property>
               <property name="text">
                <string>FPS:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="CMB_fps">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="toolTip">
                <string>FPS of the footage that will be imported by the EDL</string>
               </property>
               <item>
                <property name="text">
                 <string>23.98</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>24</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>25</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>29.97</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>30</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>50</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>59.94</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>60</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="BTN_footageDir">
             <property name="toolTip">
              <string>The base directory where footage referenced by the EDL will be searched for</string>
             </property>
             <property name="text">
              <string>Footage root directory...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LNE_footageDir">
             <property name="toolTip">
              <string>The base directory where footage referenced by the EDL will be searched for</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mappings">
      <attribute name="title">
       <string>Mappings</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_5">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="toolTip">
            <string>A CSV file mapping input footage paths to the output EXR sequence paths</string>
           </property>
           <property name="text">
            <string>Mapping file</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="LNE_mappingFile"/>
           </item>
           <item>
            <widget class="QPushButton" name="BTN_mappingFile">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QTableWidget" name="TBL_mappings">
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="BTN_addMapping">
           <property name="toolTip">
            <string>Add another mapping entry</string>
           </property>
           <property name="text">
            <string>Add mapping</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="metadata">
      <attribute name="title">
       <string>Metadata</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="toolTip">
              <string>Metadata that will be embedded into the EXR sequence. Name is the metadata name, value is what that metadata variable is set to.</string>
             </property>
             <property name="text">
              <string>Metadata</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BTN_addMeta">
             <property name="toolTip">
              <string>Add another metadata entry</string>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QTableWidget" name="TBL_metadata">
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="options">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_7">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QCheckBox" name="CHK_downscale">
         <property name="toolTip">
          <string>All incoming footage will be reformatted to the provided dimensions prior to exporting to an EXR sequence</string>
         </property>
         <property name="text">
          <string>Downscale</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="CHK_proxy">
         <property name="toolTip">
          <string>In addition to the EXR sequence, export another image sequence of the given type</string>
         </property>
         <property name="text">
          <string>Proxy sequence</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="LBL_proxyDownscale">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Additional downscaling for the proxy image sequence</string>
             </property>
             <property name="text">
              <string>Scale:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CMB_proxyDownscale">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Additional downscaling for the proxy image sequence</string>
             </property>
             <item>
              <property name="text">
               <string>100%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>75%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>50%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>25%</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="CHK_proxyToSubdir">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>By default, the proxy image sequence will export in the same directory as the EXR sequence. This provides the ability to specify a path to a subdirectory within the EXR sequence's directory.</string>
           </property>
           <property name="text">
            <string>Export to subdirectory</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="LNE_proxySubdir">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>The path for the subdirectory where the proxy sequence will be exported to. This is relative to where the EXR sequence is being exported to.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="LBL_proxyFormat">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>File type for the proxy image sequence</string>
             </property>
             <property name="text">
              <string>File type:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CMB_proxyFormat">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>File type for the proxy image sequence</string>
             </property>
             <item>
              <property name="text">
               <string>jpeg</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>png</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>targa</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>tiff</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="CHK_proxyNameAppend">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>By default, the proxy sequence will have the same name as the EXR sequence (aside from the extension). Enable this a type in the field what should be added to the end of the name</string>
           </property>
           <property name="text">
            <string>Append to file name</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="LNE_proxyNameAppend">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>_PROXY</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="CMB_downscale">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>75%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>66%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>50%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>33%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25%</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="CHK_colorspace">
         <property name="text">
          <string>Set input colorspace</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="toolTip">
          <string>This will get executed in a shell once for every shot, {SHOT} is available as a variable.</string>
         </property>
         <property name="text">
          <string>Post-write script</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="LNE_script"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
